packages/bluesnap-direct-integration/src/BlueSnapV2PaymentMethod.test.tsx

```diff
@@ -1,57 +1,75 @@
+/* eslint-disable @typescript-eslint/no-unsafe-assignment */
```

Comment:
It's ok to set this rule in eslint config for all test.tsx files in your package